chore: use values.yaml template for hmac

dali546 opened the pull request:
added `optional` incorrectly

dali546 committed:
use values

rawlingsj committed:
Merge branch 'main' into fix/hmac-secret

vbehar commented on charts/lighthouse-webui-plugin/values.yaml:
```diff
@@ -25,6 +25,7 @@ secrets:
       secretKeyRef:
         name: lighthouse-hmac-token
         key: hmac
+        optional: true
```
I think that instead of making it optional for everybody, you should make it optional only for bitbucket users, by doing similar to https://github.com/jenkins-x/jx3-versions/blob/master/charts/jx3/lighthouse/values.yaml.gotmpl#L60-L63 in https://github.com/jenkins-x/jx3-versions/blob/master/charts/jx3/lighthouse-webui-plugin/values.yaml.gotmpl

dali546 committed:
remove optional key

don't make hmac optional by default, as other git providers still use
it.
instead it will be added in the versionStream

dali546 commented:
related: https://github.com/jenkins-x/jx3-versions/pull/2694

vbehar commented:
/ok-to-test
